.github/workflows/main-push-autotag.yml

```diff
@@ -0,0 +1,68 @@
+name: main push autotag
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  auto-git-tag:
+    runs-on: ubuntu-latest
+
+    # Expose step outputs as job outputs
+    outputs:
+      new_tag: ${{ steps.run_git_auto_tag.outputs.new_tag }}
+
+    steps:
+    - uses: actions/github-script@v6
+      id: get_pr_data
+      with:
+        script: |
+            return (
+              await github.rest.repos.listPullRequestsAssociatedWithCommit({
+                commit_sha: context.sha,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+              })
+            ).data[0];
+
+    - uses: actions/checkout@v2
+    - uses: fregante/setup-git-user@v2
+    - name: Run Auto-tagging
+      id: run_git_auto_tag
+      # User controlled input needs to be santitised beforehand e.g., by adding an
+      # intermediate env var to prevent the workflow being exposed to a critical
+      # command injection attack
+      env:
+        PR_BODY: '${{ fromJson(steps.get_pr_data.outputs.result).body }}'
+        PR_NUMBER: '${{ fromJson(steps.get_pr_data.outputs.result).number }}'
+      run: |
+        echo "Attempting to auto-tag merge commit"
+        new_tag=$(.github/auto-tag.sh "${PR_BODY}" "${PR_NUMBER}")
+        echo "new_tag=${new_tag}"  >> "$GITHUB_OUTPUT"
```

Comment:
In the script its referenced in caps: `${NEW_TAG}`

Reply:
See the `outputs` section of this job :)